utils/scripts/install-ubuntu20.sh

```diff
@@ -1,23 +1,20 @@
-#!/bin/sh
+#!/bin/sh -e
```

Comment:
Same as above, the "-e" and other options are ignored on shebang lines processed by bash.